base/src/main/java/org/geysermc/api/GeyserApiBase.java

```diff
@@ -115,16 +116,27 @@ public interface GeyserApiBase {
     String usernamePrefix();
 
     /**
-     * Returns the major API version. Bumped whenever a significant breaking change or feature addition is added.
+     * @deprecated in favor of {@link #baseApiVersion()}.
      */
+    @Deprecated
```

Comment:
`forRemoval = true` and so forth with other deprecations until we get jetbrains annotations. maybe we add that library now.